File: src/main/java/net/allape/bus/Services.java
package net.allape.bus;

import com.intellij.notification.*;
import com.intellij.openapi.ui.MessageType;
import com.intellij.openapi.util.text.StringUtil;
import com.intellij.openapi.wm.ToolWindow;

public final class Services {

    /**
     * message中用到的group
     */
    public static final String GROUP = "xftp";

    /**
     * 当前tool window
     */
    public static ToolWindow TOOL_WINDOW = null;

    /**
     * 消息提醒
     * @param message 提示的消息
     */
    public static void message (String message, MessageType type) {
        NotificationGroup notificationGroup = NotificationGroupManager.getInstance().getNotificationGroup(GROUP);
        Notification notification = notificationGroup.createNotification(message, type);
        Notifications.Bus.notify(notification);
    }
}

File: src/main/kotlin/net/allape/common/Services.kt
package net.allape.common

import com.intellij.notification.NotificationGroupManager
import com.intellij.notification.Notifications
import com.intellij.openapi.ui.MessageType
import com.intellij.openapi.wm.ToolWindow

class Services {

    companion object {

        lateinit var toolWindow: ToolWindow

        /**
         * message中用到的group
         */
        const val GROUP = "xftp"

        /**
         * 消息提醒
         * @param message 提示的消息
         */
        fun message(message: String, type: MessageType = MessageType.WARNING) {
            val notificationGroup =
                NotificationGroupManager.getInstance().getNotificationGroup(GROUP)
            val notification = notificationGroup.createNotification(
                message,
                type
            )
            Notifications.Bus.notify(notification)
        }

    }

}